modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -257,11 +257,18 @@ public Response wipeData(@HeaderParam("Accept") String acceptHeader,
 		Message message = new Message();
 
 		try {
-			CommandOperation operation = new CommandOperation();
+			WipeData wipeData = wipeDataBeanWrapper.getOperation();
+
+			if (wipeData == null) {
+				throw new OperationManagementException("WipeData bean is empty");
```

Comment:
It should be better to log the errors here rather than just throwing (To facilitate better debugging)